src/main/java/nl/tudelft/contextproject/model/level/util/Vec2I.java

```diff
@@ -0,0 +1,118 @@
+package nl.tudelft.contextproject.model.level.util;
+
+/**
+ * 2d integer vector class.
+ * Used for locations of objects of importance on the map.
+ * Mainly used in map generation.
+ */
+public class Vec2I {
+	
+	/**
+	 * The x coordinate of the vector.
+	 */
+	public int x;
```

Comment:
I discussed this with @Taeir . We came to the conclusion that in this case single letter fields are fine, as this letters accurately represent the meaning of the vector. JMonkey vectors are defined in the same way, so it part ease of use.